### PR TITLE
added a table

### PR DESCRIPTION
**Pull request by jack99999**
second attempt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,8 @@ and some list items
   * first
   * second
   * third
+  
+  | this      | is | a    | table |
```
Comment (brynhs) [inline README.md:12]: This is fascinating

File: README.md [hunk 2]
```diff
@@ -8,3 +8,8 @@ and some list items
...
+  |:----------|:---|:-----|:------|
+  | something | in | here | too   |
```
Comment (brynhs) [inline README.md:14]: This is also fascinating

Comment (jack99999) [reply README.md:14]: ok, got those. somehow i think i can make another change and then update the pull request... will try...

ta

Comment (brynhs) [reply README.md:14]: If you make another commit on this branch and push it then it should update the pull request. At least that's how it works on our system (which isn't Github)

Comment (jack99999) [reply README.md:14]: just done that. hope it worked

Comment (jack99999) [reply README.md:14]: but now it says i want to merge 2 commits. i thought it would be just one.

Comment (jack99999) [reply README.md:14]: ah, no. getting mixed up.. yes, should be 2 commits.

Comment (brynhs) [reply README.md:14]: You might be able to amend the previous commit somehow but I'd make a new one if you want further changes

**Review by brynhs — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jack99999) [reply README.md:14]: i made another commit (and push to get it to github) and github automatically updated the same pull request.

**Pull request merged**